geonotebook/utils.py

```diff
@@ -13,3 +13,6 @@ def get_kernel_id(kernel):
     connection_file_path = ipykernel.get_connection_file()
     connection_file = os.path.basename(connection_file_path)
     return connection_file.split('-', 1)[1].split('.')[0]
+
+def transform_coordinates(source_srs, target_srs, x, y):
```

Comment:
@dorukozturk we probably want to add a array based as well as that would be more efficient, passing one coordinate at a time would be slow if we want to have it as general util.